set_core_version.py

```diff
@@ -55,6 +55,7 @@ def replace_toml_version(relpath, newversion):
     replace_toml_version("Cargo.toml", newversion)
     replace_toml_version("deltachat-ffi/Cargo.toml", newversion)
 
+    subprocess.call(["cargo", "check"])
```

Comment:
this was what i typically missed on previous tagging -- updating Cargo.toml is not enough -- Cargo.lock also contains versions.

Reply:
cool it's added now :)